app/src/pages/inside/testCaseLibraryPage/testCaseLibraryPage.tsx

```diff
@@ -91,7 +102,7 @@ export const TestCaseLibraryPage = () => {
                     icon={Parser(ImportIcon as unknown as string)}
                     data-automation-id="importTestCase"
                     adjustWidthOn="content"
-                    onClick={openImportFolderModal}
+                    onClick={() => openImportFolderModal({ folderName: currentFolderName ?? '' })}
```

Comment:
I think here we do extra work that can be omitted if we adjust `openImportFolderModal` type to accept `folder` either as `string` or `underfined`

```suggestion
                    onClick={() => openImportFolderModal({ folderName: currentFolderName })}
```